portfolio/src/main/webapp/index.html

```diff
@@ -5,7 +5,7 @@
     <title>Nateyana Blake Portfolio Site</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="css/styles.css" type="text/css" />
-    <link rel="stylesheet" href="css/bootstrap-grid.css" type="text/css" />
+    <!-- <link rel="stylesheet" href="css/bootstrap-grid.css" type="text/css" /> -->
```

Comment:
Could you add a reason to remove this line in the commit description or here..?

Reply:
I wanted to see if my layout would break when I removed it